hack/verify-latest-onprem-config.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+make generate-onprem-environment
```

Comment:
Instead of invoking `make` here, I think you should simply add these as dependencies to the `verify-latest-onprem-config` target.

Reply:
Done!